nvme-print.c

```diff
@@ -3165,6 +3166,17 @@ static void nvme_show_id_ctrl_nwpc(__u8 nwpc)
 	printf("\n");
 }
 
+static void nvme_show_id_ctrl_ocfs(__u16 ocfs)
+{
+	__u16 rsvd = (ocfs & 0xfffe) >> 1;
+	__u8 copy_fmt_0 = ocfs & 0x1;
+	if (rsvd)
+		printf("  [15:1] : %#x\tReserved\n", rsvd);
+	printf("  [0:0] : %#x\tController %s Copy Format 0h\n",
+		copy_fmt_0, copy_fmt_0 ? "Supports" : "Doesn't Supports");
```

Comment:
I think the style is "Controller Copy Format 0h (Supported|Not Supported)".

Reply:
yeah, right modified @birkelund , thanks!